app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cached_data = {}
+
+    def wrapper(*args) -> any:
+        if args in cached_data:
```

Comment:
Why you check only `args`?
What if function takes `kwargs`?

Reply:
but the dict is not hashed, and there will be a TypeError when I try to use the dict as a key, how can I save it?